gdal/doc/source/api/index.rst

```diff
@@ -89,7 +89,16 @@ API
 
    `Java API <../java/index.html>`_
    --------------------------------
-   
+
+
+   GDAL/OGR In Other Languages
+   ++++++++++++++++++++++++++++
+
+   There is a set of generic `SWIG <http://www.swig.org/>`__ interface files in the GDAL source tree (subdirectory swig) and a set of language bindings based on those. Currently active ones are 
+   `CSharp <https://trac.osgeo.org/gdal/wiki/GdalOgrInCsharp/>`__, `Java <https://trac.osgeo.org/gdal/wiki/GdalOgrInJava>`__, `Perl <https://trac.osgeo.org/gdal/wiki/GdalOgrInPerl>`__, and `Python <https://trac.osgeo.org/gdal/wiki/GdalOgrInPython>`__.
```

Comment:
ideally we should remove any link to the trac instance and move the content here., but that would be a larger task

Reply:
there should be a mention that the Perl ones are going to be removed in GDAL 3.5 and that the below mentioned Geo-GDAL-FFI will be the replacement

Reply:
@rouault, for perl case: 

- I can add a warning box and put the information about Perl removal, something like _Since GDAL 3.5 Perl Gdal bindings will be replaced by Geo-GDAL-FFI_
- For the other links, I can create new sub-pages (one rst file for each language) and migrate the text but what is the right place? Can be in here?

![image](https://user-images.githubusercontent.com/4480601/110244997-49afb180-7f59-11eb-8e0b-25354e06a8e2.png)

Reply:
@rouault If you agreed with this preview, I can migrate the pages of languages bindings.

![image](https://user-images.githubusercontent.com/4480601/110246289-fb9dac80-7f5e-11eb-9c93-b20efcef9f73.png)

Reply:
>     * For the other links, I can create new sub-pages (one rst file for each language) and migrate the text but what is the right place? Can be in here?

I let you decide how to organize that, but a critical look is needed since the wiki content might be outdated. And for, Java and Python, the trac page content should probably be moved into the what exists in https://gdal.org/api/ , or the way round